chart/templates/mayastor/io/io-engine-daemonset.yaml

```diff
@@ -130,21 +108,49 @@ spec:
           mountPath: /dev/shm
         - name: configlocation
           mountPath: /var/local/{{ .Release.Name }}/io-engine/
-        - name: hugepage
-          mountPath: /dev/hugepages
+        - name: hugepages-2mi
+          mountPath: /dev/hugepages-2mi
+        {{- if .Values.io_engine.resources.requests.hugepages1Gi }}
+        - name: hugepages-1gi
+          mountPath: /dev/hugepages-1gi
+        {{- end }}
         resources:
           limits:
             cpu: {{ .Values.io_engine.resources.limits.cpu | default (include "coreCount" .) | quote }}
             memory: {{ .Values.io_engine.resources.limits.memory | quote }}
             hugepages-2Mi: {{ .Values.io_engine.resources.limits.hugepages2Mi | quote }}
+            hugepages-1Gi: {{ .Values.io_engine.resources.requests.hugepages1Gi | quote }}
           requests:
             cpu: {{ .Values.io_engine.resources.requests.cpu | default (include "coreCount" .) | quote }}
             memory: {{ .Values.io_engine.resources.requests.memory | quote }}
             hugepages-2Mi: {{ .Values.io_engine.resources.requests.hugepages2Mi | quote }}
+            hugepages-1Gi: {{ .Values.io_engine.resources.requests.hugepages1Gi | quote }}
         ports:
         - containerPort: 10124
           protocol: TCP
           name: io-engine
+      {{- if .Values.base.metrics.enabled }}
+      - name: metrics-exporter-io-engine
```

Comment:
How were metrics being exported if this section didn't exist?

Reply:
it did exist, these bits are just being reordered: https://github.com/openebs/mayastor-extensions/pull/712/commits/973bacbe92b9643c8606403847614b3e2af15bc1

Reply:
How did I miss that !!